### ISSUE TITLE
Add CloudCannon CMS support

### ISSUE THREAD

**Issue by markdumay** [gethinode/hinode#322]
**Is your feature request related to a problem? Please describe.**
Currently Hinode supports direct editing of markdown content only. Although this is adequate for most developers, it is less friendly for less tech-oriented users.

**Describe the solution you'd like**
Visual Content Managements Systems (CMSes) provide a more intuitive, user-friendly approach for content management. In general, so-called [headless CMSs](https://jamstack.org/headless-cms/) are either Git-based or API-driven. In a Git-based approach, the content still resides within the same repository and does not require an additional back-end or database.

CloudCannon is one of the more popular and robust Git-based CMSes. Ideally, Hinode would offer out-of-the-box integration with the CMS. The integration should include a basic configuration, documentation, and support for custom shortcodes.

**Describe alternatives you've considered**
Two other popular Git-based CMSes compatible with Hugo are [Decap CMS](https://decapcms.org) and [Tina CMS](https://tina.io). At the time of writing, Decap CMS is being transferred to a new owner and still requires significant code changes (the [latest release ](https://github.com/decaporg/decap-cms/releases/tag/netlify-cms%402.10.192)is from April 2022). Although promising, Tina CMS is still a relatively new product that appears less robust compared to CloudCannon. However, additional support for these alternative CMSes could be considered later.

**Comment by myrthos** [comment]
That would be really interesting. The only reason I see to adding Tina or Decap next to CloudCannon is that they are free or have a free plan :)

**Comment by markdumay** [comment]
Yes, I agree - it's a pity CloudCannon does not have a free plan (for public repositories).
